app/serializers/permitted_params_serializer.rb

```diff
@@ -5,7 +5,7 @@ def object
     PermittedParams.new
   end
 
-  %w[user vote motion membership_request membership
+  %w[user vote motion proposal membership_request membership
```

Comment:
We tripped on this thing again, heh. I never would have imagined how many times it would crop up and be a  pain :sheep: